### PR TITLE
#33 Sort collaboration partners by alphabetic order

### PR DESCRIPTION
**Pull request by victorvicari**
Updated the gradle version. In the view model `CollaborationsViewModel`, added sort by the collaborator'
s name. Every name, set `trimStart()` to remove any possible space before the first letter. And `capitalize()`, to capitalize the names.

sort_collab_list

### CODE REVIEW

File: app/src/main/java/com/wtmberlin/ui/CollaborationsViewModel.kt [hunk 1]
```diff
@@ -31,8 +31,9 @@ class CollaborationsViewModel(repository: Repository) : ViewModel() {
 
     private fun processVenues(list: List<VenueName>) {
         adapterItems.value = list.map {
-            CollaborationsAdapterItem(it.name, it.name)
+            CollaborationsAdapterItem(it.name, it.name.trimStart().capitalize())
```
Comment (Jedyny) [inline app/src/main/java/com/wtmberlin/ui/CollaborationsViewModel.kt]: Maybe let's not capitalize venue names, some of them may start from a lower-case letter by purpose. If you see a mistake maybe it's better to fix it in Meetup.

File: build.gradle [hunk 2]
```diff
@@ -9,7 +9,7 @@ buildscript {
     ext.constraintLayoutVersion = "1.1.3"
     ext.coreVersion = "1.0.1"
     ext.fragmentVersion = "1.0.0"
-    ext.gradleVersion = '3.3.1'
+    ext.gradleVersion = '3.4.1'
```
Comment (Jedyny) [inline build.gradle]: Could you bump the version of Gradle in a separate MR?

Comment (victorvicari) [reply build.gradle]: Sure. I'll make the change

File: app/src/main/java/com/wtmberlin/ui/CollaborationsViewModel.kt (hunk 1 above)
Comment (victorvicari) [reply app/src/main/java/com/wtmberlin/ui/CollaborationsViewModel.kt]: I understand. I'll do the proper modifications.

**Review by Jedyny — APPROVED** [review]

**Pull request merged**